lib/literature/router.ex

```diff
@@ -149,7 +149,7 @@ defmodule Literature.Router do
   defmacro literature(path, opts \\ []) do
     opts = Keyword.put(opts, :application_router, __CALLER__.module)
 
-    routes = Keyword.get(opts, :only, ~w(index tags authors show)a)
+    routes = Keyword.get(opts, :only, ~w(index index_pages tags authors show)a)
```

Comment:
@rcmonsayac how does this work exaclty , what is index_page as i can not see it anywhere else in the code

Reply:
it is actually just a flag if we want to enable pagination for the index like this

```
if :index_pages in routes do
     live("/page/:page", BlogLive, :index, route_opts)
end
```

This is for single index pages like for the support where we do not need pagination